### PR TITLE
REMOVES SHIELF

### PR DESCRIPTION
**Pull request by EgoSumStultus**
## About The Pull Request
REMOVES SHIELF
## Why It's Good For The Game
SHIELF MAN BAD
## Changelog
:cl:
fix: FIXED SHIELF
/:cl:

### CODE REVIEW

**Comment by Sishen1542** [comment]
bro its a scabbard

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -414,8 +414,8 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 /datum/uplink_item/dangerous/rapier
 	name = "Rapier"
 	desc = "A fancy rapier with a diamond tip piercing anything that it comes into contack with. \
-			The rapier comes with its own shielf, this is rather noticeable as only the captain is known to carry a shielf. \
-			The shielf itself can be used to block melee attacks only. Its also jet black colours."
+			The rapier comes with its own sheath, this is rather noticeable as only the captain is known to carry a sheath. \
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm:417]: You miss spelled Shielf 4 times in a row man

Comment (Linzolle) [reply code/modules/uplink/uplink_items.dm:417]: I don't think shielf is a word

Comment (Trilbyspaceclone) [reply code/modules/uplink/uplink_items.dm:417]: IT IS DAMN IT

**Pull request merged**